src/bridge-deployer/src/cli.rs

```diff
@@ -68,6 +69,16 @@ pub struct Cli {
         help_heading = "Display"
     )]
     quiet: bool,
+
+    /// Custom path to the canister_ids.json file.
+    ///
+    /// If not provided, the default path is used.
```

Comment:
Use clap to set the default value so it will appear in the help

Reply:
Unfortunately this can't be done, since the path is different based on the evm network